regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -46,6 +47,16 @@ var PreambleView = ChildView.extend({
     );
   },
 
+  openCitation: function(e) {
+    var $target = $(e.currentTarget);
+    var hash = $target.attr('href');
```

Comment:
This works right now, but I think it's due to a bug in the underlying markup. The href _should_ be including more than just a hash (consider, for example, the url associated with III.B when you're in I.A). Do you mind accounting for that now?